onmt/translate/translator.py

```diff
@@ -231,7 +232,7 @@ def from_opt(
             report_score (bool) : See :func:`__init__()`.
             logger (logging.Logger or NoneType): See :func:`__init__()`.
         """
-
+        # TODO: maybe add dynamic part
```

Comment:
Is it doable without making things too complicated?

Reply:
That involves handling tgt=None for Transforms and potential fix for server also. A little bit complex I would say. Maybe do this when we get the time.